scout/server/extensions/clinvar_extension.py

```diff
@@ -17,7 +17,7 @@ class ClinVarApi:
 
     def __init__(self):
         self.convert_service = "/".join([PRECLINVAR_URL, "csv_2_json"])
-        self.validate_service = "/".join([PRECLINVAR_URL, "validate"])
+        self.validate_service = "/".join([PRECLINVAR_URL, "dry-run"])
```

Comment:
Good! Does this work as expected now with actually retrieving a new SUB-number?

Reply:
It should, but let's try it!